### ISSUE TITLE
BOM

### ISSUE THREAD

**Issue by xk4l23** [putyn/nixie-clock#2]
Hi, this is really a great project, would like to build one for me, too. Is there any change to get the BOM to the schematic? Or did I miss it? If so, please point me to the right place.

**Comment by fabicodes** [comment]
Yeah, you're right. Although you can even create a BOM there aren't any values specified

**Comment by xk4l23** [comment]
That is what I am asking about. The values for the parts.

**Comment by putyn** [comment]
looks like I'm a lousy repo owner, I don't have notifications turned on. @xk4l23 are you still interested in the BOM?
